e2e-tests/insurance/cypress/e2e/journeys/your-buyer/alternative-currency/alternative-currency-page.spec.js

```diff
@@ -97,23 +101,81 @@ context('Insurance - Your Buyer - Alternative currency - As an exporter, I want
   });
 
   describe('form submission', () => {
-    describe('when submitting an empty form', () => {
+    describe('when selecting the alternative currency radio but not entering an alternative currency via the autocomplete ', () => {
       beforeEach(() => {
         cy.navigateToUrl(url);
       });
 
       it('should render validation errors', () => {
-        cy.clickSubmitButton();
+        rendersAlternativeCurrencyValidationError();
+      });
+    });
+
+    describe('when submitting a supported currency', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+      });
+
+      describe(EUR.name, () => {
+        const { isoCode } = EUR;
```

Comment:
For all the ISO codes in this test, could just import `EUR_CURRENCY_CODE`, `GBP_CURRENCY_CODE` etc

Reply:
I am using the name too though?

Reply:
Only for the test description though right?

Reply:
yes, should I hard type the name like euros?

Reply:
I was thinking could just use the ISO code for all, e.g `EUR_CURRENCY_CODE`,  what do you think?